cypress/integration/like.spec.ts

```diff
@@ -30,3 +30,91 @@ it('likes an episode then finds it in the library', () => {
     });
   });
 });
+
+it.skip('can remove the like from a liked episode', () => {
+  cy.createSupabaseClient().then(async (supabase) => {
+    const { data: podcast } = await supabase
+      .from<SBPodcast>('podcasts')
+      .select('id, episodes (title)')
+      .limit(1)
+      .single();
+    expect(podcast).to.be.not.null;
+    if (podcast === null) return;
+    const titles = podcast.episodes.map(({ title }) => title);
+    cy.visitAndWaitForHydration(`/podcasts/${podcast.id}`);
+
+    // Like all the episodes
+    cy.findAllByTestId('episode-card').each((card, index) => {
+      cy.wrap(card).within(() => {
+        cy.findByText(titles[index]).should('exist');
```

Comment:
issue: this doesn't really test that this title is linked as an accessible name of the episode card

suggestion: I think the test would be better if we were to loop over the titles and for each title `findByRole('article', { name: title })`. That way we test that each episode has an accessible name, and also it allows us to immediately chain a `.within()` off of it

Reply:
This is inside `cy.wrap(card)`, which is already the `<article>`. I'm just verifying that it has the text inside the card. Also, `{ name: title }` would check for an accessible [`title` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title), not text content.

Reply:
Yeah, but it's not enough to have text inside the card for assistive technologies to recognize that this text is the accessible name of the entire card. I've implemented this using `aria-labelledby`.

`{ name: title }` would first check for an accessible name coming from `aria-labelledby`, then `aria-label`, and only then the `title`

Reply:
Ah right, it was failing only because it's inside the `within`. Actually, now that you point it out, I think that "_to recognize that this text is the accessible name of the entire card_" is not the responsibility of this test and this line should probably be removed altogether, right?

Reply:
Yeah, perhaps it's not